### DISCUSSION TITLE
Accesso tramite https

### DISCUSSION THREAD

**Discussion by MarcoBanchieri** [indomus/forum#348] [Networking]
Buonasera a tutto il gruppo. Premetto che non sono assolutamente pratico di Linux,  che sto studiando da poco Home Assistant. Mi piacerebbe accedere al raspberry  tramite https anche perchè ho letto che per poter gestire Chromecast è necessario l'accesso protetto. Ho attivato SSH &Web terminal, perchè pensavo fosse utile, ma forse ho capito male io. E' possibile attivare l'https ? Come ? Visto che non si può oltretutto accedere al sistema Linux ..... Grazie

**Comment by snakuzzo** [comment]
https://indomus.it/guide/collegarsi-da-remoto-a-home-assistant-installato-con-distribuzione-hassio/
